packages/headless/src/state/state-sections.ts

```diff
@@ -339,5 +339,5 @@ export interface DocumentSuggestionSection {
   /**
    * The document suggestions for case assist.
    */
-  documentSuggestions: DocumentSuggestionState;
+  documentSuggestion: DocumentSuggestionState;
```

Comment:
@nathanlb This interface needed to be updated like `CaseInputState` and `CaseFieldState`.